### PR TITLE
Update onboarding document

### PR DESCRIPTION
**Pull request by idazucchi**
The document was already good, I've updated a few links and references.
Please add anything you think it's missing!

See [#608](https://github.com/ebi-ait/hca-ebi-wrangler-central/issues/608)

### CODE REVIEW

File: docs/ebi-wrangler-onboarding.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ See [chairing rota](https://docs.google.com/spreadsheets/d/1-y7QNoIdwYdFHVxJUc5X
 | Meeting                              | Day       | Time   | Frequency             | Notes                                |
 |--------------------------------------|-----------|--------|-----------------------|--------------------------------------|
 | Operations planning/review           | Wednesday | 9.30am | Weekly                | |
-| Dev sprint kick off meeting          | Wednesday | 10.30am| Bi-weekly             | At start of EBI sprint,only dev wrangler needs to attend |
+| Dev sprint kick off meeting          | Wednesday | 10.30am| Bi-weekly             | At start of EBI sprint, only dev wrangler needs to attend |
```
Comment (ESapenaVentura) [inline docs/ebi-wrangler-onboarding.md:123]:
```suggestion
| Dev sprint kick off meeting          | Wednesday | 10.30am| Bi-weekly             | At start of EBI sprint, only wrangler on development needs to attend |
```

File: docs/ebi-wrangler-onboarding.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ See [chairing rota](https://docs.google.com/spreadsheets/d/1-y7QNoIdwYdFHVxJUc5X
...
 | EBI-DCP Sprint Planning & Demo       | Tuesday   | 2pm    | Bi-weekly             | 1 hour, telecon, at end of a sprint  |
 | EBI DCP Sprint Retro                 | Tuesday   | 4pm    | Bi-weekly             | 1 hour, telecon                      |
 | User stories for next sprint         | Monday    | 10am   | Bi-weekly             | 1 hour, monday before end of EBI sprint |
```
Comment (ESapenaVentura) [inline docs/ebi-wrangler-onboarding.md:126]:
```suggestion
```
Don't think this is happening anymore

File: docs/ebi-wrangler-onboarding.md [hunk 3]
```diff
@@ -147,7 +147,7 @@ Useful channels include:
 
 [#hca-development](https://embl-ebi-ait.slack.com/archives/C016R78CDV1) - channel for discussion about the current development work, sorting testing, answering questions etc
 
-#hca-wrangler-metadata - this is where most of the discussion relating to wrangling and the the HCA metadata are discussed
+#hca-wrangler-metadata - this is where most of the discussion relating to wrangling and the HCA metadata are discussed
 
 [#dcp-2](https://embl-ebi-ait.slack.com/archives/C01360XN04S) - This is a shared channel with the HCA slack. It is used for general announcements about the DCP or questions about the DCP, this is a good place to introduce yourself when you have logged in.
```
Comment (ESapenaVentura) [inline docs/ebi-wrangler-onboarding.md:153]:
```suggestion

[#dcp-ops](https://embl-ebi-ait.slack.com/archives/C9XD6L0AD) - This is a shared channel with the HCA slack. It is used for operation discussion/announcements, including when releases are available for review or they go live.

```

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]
Added some comments :)

File: docs/ebi-wrangler-onboarding.md (hunk 2 above)
Comment (gabsie) [reply docs/ebi-wrangler-onboarding.md:126]: It is happening between me and Amnon (for now). No need to have it in this doc, certainly.

**Review by gabsie — APPROVED** [review]
Looks good to me!

File: docs/ebi-wrangler-onboarding.md [hunk 4]
```diff
@@ -302,5 +301,4 @@ Repositories you will mainly use:
 ### Miscellanea
 
 - Printers: If you install the correspondent apps (You can search for them as “Printer”, under the “Managed software centre” app), you will be able to use the Konica printers in both buildings. Using them to print personal photocopies has a little cost (1 pence) that is automatically subtracted from your salary.
```
Comment (Wkt8) [inline docs/ebi-wrangler-onboarding.md:303]:
```suggestion
- Printers: If you install the correspondent apps (You can search for them as “Printer”, under the “Managed software centre” app), you will be able to use the Konica printers in both buildings. Using the printers has a small cost which comes out of the group's budget.  
```

**Review by Wkt8 — CHANGES_REQUESTED** [review]
Looks good! Added a small change to the printing section.

**Comment by idazucchi** [comment]
I've applied the suggestions you made and I'm ready to merge to master

**Pull request merged**